pyramid-app/{{ cookiecutter.slug }}/{{ cookiecutter.package_name }}/app.py

```diff
@@ -1,9 +1,23 @@
+{% if cookiecutter.get("postgres") == "yes" %}
+from os import environ
+
+{% endif %}
 from pyramid.config import Configurator
 from pyramid.view import view_config
 
 
 def create_app(_=None, **settings):
     with Configurator(settings=settings) as config:
+{% if cookiecutter.get("postgres") == "yes" %}
+        config.registry.settings["database_url"] = environ["DATABASE_URL"]
+
+        config.registry.settings["tm.annotate_user"] = False
+        config.include("pyramid_tm")
+
+        config.include("{{ cookiecutter.package_name }}.models")
+        config.include("{{ cookiecutter.package_name }}.db")
+
+{% endif %}
```

Comment:
Once you've written `db.py` this is all you have to do to integrate the SQLAlchemy session and models into the Pyramid request lifecycle. `pyramid_tm` creates a new transaction for each request and commits or aborts the transaction at the end of each request